<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gBoxA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>执行板串口[19200 - RS485]</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="gBoxAWid" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>411</width>
      <height>110</height>
     </rect>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>执行板：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBoxC1">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>校准模式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBoxC2">
         <item>
          <property name="text">
           <string>一键校准模式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>继电器轮流开关模式</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="minAEdit">
         <property name="text">
          <string>5.5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>----</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxAEdit">
         <property name="text">
          <string>5.6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="minWEdit">
         <property name="text">
          <string>130</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>KW</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>----</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxWEdit">
         <property name="text">
          <string>133</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>KW</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxB">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>461</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>标准源串口[9600 - RS232]</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QWidget" name="gBoxBWid" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxG">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>261</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>统计显示</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="sumLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>总数：0  通过：0  失败：0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startOrStop">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>67</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gBoxD">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>当前工位</string>
    </property>
    <widget class="QPushButton" name="resBtu">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>140</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>重置工位</string>
     </property>
    </widget>
    <widget class="QLabel" name="station">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>XXXXX</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gBoxH">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1161</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>显示区</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>总和</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>开关状态</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>电流（A）</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>电压（V)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>功率（KW）</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>开关</string>
        </property>
       </column>
       <item row="0" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>50000</width>
           <height>70</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>60</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>MPDU自动校准工具（R）</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>历史结果：</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gBoxF">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>输出结果</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="PassLabel">
       <property name="font">
        <font>
         <pointsize>70</pointsize>
        </font>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>161</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>标准源复位测试（220|6）</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gBoxI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>操作时间：</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>上次操作时间：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="timeLast">
       <property name="text">
        <string>xxx</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLabel" name="timeFirst">
       <property name="text">
        <string>xxx</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>本次操作时间：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
